sktime/classification/kernel_based/_arsenal.py

```diff
@@ -462,12 +462,17 @@ def get_test_params(cls, parameter_set="default"):
             ``create_test_instance`` uses the first (or only) dictionary in ``params``.
         """
         if parameter_set == "results_comparison":
-            params = {"num_kernels": 20, "n_estimators": 5}
+            return {"num_kernels": 20, "n_estimators": 5}
         else:
-            params = {
+            params1 = {
+                "max_dilations_per_kernel": 30,
+                "rocket_transform": "minirocket",
+                "n_jobs": -1,
+            }
+            params2 = {
                 "num_kernels": 10,
                 "n_estimators": 2,
                 "save_transformed_data": True,
```

Comment:
not blocking, but it would be great if we could also change th eother parameters:

```python
        num_kernels=2000,
        n_estimators=25,
        rocket_transform="rocket",
        max_dilations_per_kernel=32,
        n_features_per_kernel=4,
        time_limit_in_minutes=0.0,
        contract_max_n_estimators=100,
        save_transformed_data=False,
```

also `num_kernels` should always be set and low, or the runtime will be too high.

Reply:
plus, for consistency across versions, I would leave the former `params` to now be `params1`.